.gitignore

```diff
@@ -0,0 +1,22 @@
+# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+
+# Crash log files
+crash.log
+
+# Ignore any .tfvars files that are generated automatically for each Terraform run. Most
+# .tfvars files are managed as part of configuration and so should be included in
+# version control.
+#
+# example.tfvars
```

Comment:
Yeah, it is kinda just there as a placeholder. I tried to use https://github.com/github/gitignore/blob/master/Terraform.gitignore as a base. In the other project I ignored `*.tfvars` because it actually has `.tfvars` files, whereas this one shouldn't.